[DOCS] Blocklist (#1811)
elastic/security-docs: docs/getting-started/security-ui.asciidoc

[role="screenshot"]
image::management/admin/images/host-isolation-exceptions-ui.png[Shows the Host isolation exceptions page]

[float]
[[blocklist-page]]
=== Blocklist page

> The Blocklist page allows you to prevent specified applications from running if you're certain they should never run in a host environment, extending the list of processes that {endpoint-sec} considers malicious. Refer to <<blocklist, Blocklist>> for more information.

Author: Good point @benironside -- that was some copy I lifted from another issue or epic, but it seems unnecessary and inaccurate as you say. Tweaking this sentence to match similar wording elsewhere...
```suggestion
The Blocklist page allows you to prevent specified applications from running on hosts, extending the list of processes that {endpoint-sec} considers malicious. Refer to <<blocklist, Blocklist>> for more information.
```
